wireless/wapi/src/wireless.c

```diff
@@ -1256,14 +1256,14 @@ int wapi_scan_stat(int sock, FAR const char *ifname)
           return 1;
         }
 
-      printf("err[%d]: %s\n", errno, strerror(errno));
-    }
-  else
```

Comment:
it's wrong?

Reply:
> it's wrong?

change printf(error) to WAPI_IOCTL_STRERROR, and correct return value when ioctl return >=0.
Otherwise, when multiple threads switch, errno will be set to someone else's system call return code.